src/tls/test-server.c

```diff
@@ -585,6 +589,14 @@ test_tls_client_cert_parallel (TestCase *tc, gconstpointer data)
   pid_t pid;
   int status;
 
+  /* XXX TODO This testcase runs slowly under valgrind and sometimes
+   * fails inexplicably.  It's not likely that we're going to find leaks
+   * in the code with this test, so running it under valgrind is just
+   * costing us pain.  Disable it for now.
+   */
+  if (cockpit_test_skip_slow ())
```

Comment:
is there a way to skip this only when running under valgrind?

Reply:
That's exactly what this does.  The code for `cockpit_test_skip_slow()` looks like:

```
  if (g_getenv ("COCKPIT_SKIP_SLOW_TESTS"))
    {
      g_test_skip ("Skipping slow tests");
      return TRUE;
    }
```

with `COCKPIT_SKIP_SLOW_TESTS=1` set from `Makefile.am` for the `check-memory` rule.

Reply:
oooh ok